addons/l10n_fr/data/account_chart_template_data.xml

```diff
@@ -19,7 +19,7 @@
       <field name="property_account_income_categ_id" ref="pcg_7071"/>
       <field name="income_currency_exchange_account_id" ref="pcg_766"/>
       <field name="expense_currency_exchange_account_id" ref="pcg_666"/>
-      <field name="default_pos_receivable_account_id" ref="fr_pcg_recv_pos" />
```

Comment:
Why?  The whole idea of the PoS receivable account stuff is to have a separate receivable account for the PoS.

Reply:
Because the pos account doesn't exist anymore. We expect folks to set this account themselves. I would have let this blank if didn't crash the runbot.

Reply:
As discussed still todo. We actually need the account.

Reply:
done